iogt/static/js/iogt.js

```diff
@@ -1,12 +1,13 @@
-function validateFileUpload(fileInput) {
+function validateFileUpload(fileInput, file_size_threshold) {
   if (!fileInput.files || !fileInput.files[0])
     return true;
   else {
     var file = fileInput.files[0];
-    if (file.size >= 9437184)
-      return confirm('The file you have uploaded exceeds 9mb. This will prohibit access to the file in a low ' +
-          'bandwidth setting, may restrict feature phone access, or violate your mobile network operator ' +
-          'agreements. To reduce file size, try resizing and compressing your file. Do you want to contnue?');
+    if (Math.round(file.size / 1024 / 1024) >= file_size_threshold)
```

Comment:
I would prefer to store `file_size_threshold` as an integer number of bytes and then format/parse to/from a human-readable form.

Reply:
sure

Reply:
done